sound/soc/sof/pcm.c

```diff
@@ -404,10 +404,10 @@ static int sof_pcm_open(struct snd_pcm_substream *substream)
 
 	/* set any runtime constraints based on topology */
 	snd_pcm_hw_constraint_step(substream->runtime, 0,
-				   SNDRV_PCM_HW_PARAM_BUFFER_SIZE,
+				   SNDRV_PCM_HW_PARAM_BUFFER_BYTES,
 				   le32_to_cpu(caps->period_size_min));
 	snd_pcm_hw_constraint_step(substream->runtime, 0,
-				   SNDRV_PCM_HW_PARAM_PERIOD_SIZE,
+				   SNDRV_PCM_HW_PARAM_PERIOD_BYTES,
```

Comment:
This can btw have some effects to existing test cases as the reported constraints will change for existing topology files. Not a blocker for this patch though, but just something to watch out in CI/tests results after this is in.

Reply:
yep, but we have no choice :-)